File: sample/src/main/java/contacts/sample/cheatsheet/groups/java/UpdateGroupsActivity.java
package contacts.sample.cheatsheet.groups.java;

import android.app.Activity;

import contacts.core.ContactsFactory;
import contacts.core.entities.*;
import contacts.core.groups.GroupsUpdate;

public class UpdateGroupsActivity extends Activity {

    GroupsUpdate.Result updateGroup(Group group) {
        MutableGroup mutableGroup = group.mutableCopy();
        mutableGroup.setTitle("Bad love");

        return ContactsFactory.create(this)
                .groups()
                .update()
                .groups(mutableGroup)
                .commit();
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/groups/kotlin/UpdateGroupsActivity.kt
package contacts.sample.cheatsheet.groups.kotlin

import android.app.Activity
import contacts.core.Contacts
import contacts.core.entities.Group
import contacts.core.groups.GroupsUpdate

class UpdateGroupsActivity : Activity() {

    fun updateGroup(group: Group): GroupsUpdate.Result = Contacts(this)
        .groups()
        .update()
        .groups(
            group.mutableCopy {
                title = "Bad love"
            }
        )
        .commit()
}